food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```diff
@@ -65,8 +74,19 @@ public TruckEndpoint(TruckService truckService, RouteService routeService, Subsc
         return truckService.findTruckById(id);
     }
 
+    @PostMapping("/reindex")
+    public void reindex() throws InterruptedException {
+        indexingService.initiateIndexing();
+    }
```

Comment:
I'm not entirely sure what this does, but it sounds like something we would want done under the hood instead of making the user hit an endpoint.

Reply:
A good place to do this may be inside of the `TruckService` when a new truck is created.